openedx_events/learning/data.py

```diff
@@ -0,0 +1,84 @@
+"""
+Data attributes for events within the architecture subdomain `learning`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+from datetime import datetime
+from typing import Dict
+
+import attr
+from opaque_keys.edx.keys import CourseKey
+
+
+@attr.s(frozen=True)
+class StudentData:
+    """
+    Attributes defined for Open edX student object.
+    """
+
+    username = attr.ib(type=str)
+    email = attr.ib(type=str)
+    is_active = attr.ib(type=bool, default=True)
+    meta = attr.ib(type=Dict[str, str], factory=dict)
```

Comment:
Are the contents of `meta` important enough to be in an event? If so, is it possible to parse it out into separate fields and define those here? I think it's better to more strictly define things where we can, so people don't have to worry about what kinds of data might be in a freeform dict.